app/models/resource.py

```diff
@@ -131,7 +131,13 @@ def print_resources():
             print resource.text_descriptors
             print resource.option_descriptors
 
-
+    @staticmethod
+    def get_resources():
```

Comment:
Actually I'm confused why we need this method. `Resource.query.all()` returns a list of all the `Resource` objects.